platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImpl.java

```diff
@@ -81,12 +86,16 @@ public ImportMigrationManagerImpl(
       MigrationReport report,
       MigrationZipFile zip,
       Set<String> mandatoryMigratables,
-      Stream<? extends Migratable> migratables) {
+      Stream<? extends Migratable> migratables,
+      String currentProductBranding,
+      String currentProductVersion) {
```

Comment:
❓Since we're passing `currentProductBranding` and `currentProductVersion` to the constructor, should we also do the same with the list of supported versions instead of passing it to the `doImport()` method?

Reply:
We could 😄

Reply:
I decided not to do this. To address the comment of adding a new properties file for the migration properties (https://github.com/codice/ddf/pull/4862#discussion_r287043207), I decided to have the migration code load that properties file on import versus at system startup so that changes can be made to the properties without a system restart. I think this will improve the user experience.